azavea/tilejson.io
# Remove base layer from shared tile.json file

## Overview

The base layer sneaked back into the tile.json file after merging the opacity PR. This has been removed in this PR

Connects #69

## Testing Instructions

 * share and check the github gist to make sure that the tilejson has no base layer in it
